modules/file-system/pre-existing-network-storage/outputs.tf

```diff
@@ -25,6 +25,11 @@ output "network_storage" {
     client_install_runner = local.client_install_runner
     mount_runner          = local.mount_runner
   }
+
+  precondition {
+    condition     = local.client_install_runner == null || var.fs_type == "daos"
+    error_message = "Use fs_type=daos in blueprint while using parallelstore instance."
```

Comment:
This is somewhat cryptic message, could you please rephrase it.

Reply:
Under what conditions `local.client_install_runner == null` ? IIUC, it's always defined.

Reply:
Thanks for catching. I think this is not required.